src/servers/exporter.ts

```diff
@@ -131,7 +131,7 @@ const main = async () => {
       walletRoles[role].set(balance)
     }
 
-    business_g.set(await User.count({ title: { $exists: true } }))
+    business_g.set(await User.count({ title: { $ne: null } }))
```

Comment:
needs to have exists and $ne

Reply:
I don't think that's necessary. the query I did with `{ title: { $ne: null } }` seemed to return the right number